Frameworks/CoreText/CTFont.mm

```diff
@@ -621,12 +621,17 @@ CFArrayRef CTFontCopyFeatureSettings(CTFontRef font) {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 bool CTFontGetGlyphsForCharacters(CTFontRef font, const UniChar characters[], CGGlyph glyphs[], CFIndex count) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    bool succeeded = true;
+    FT_Face face = (FT_Face)[(UIFont*)font _sizingFontHandle];
```

Comment:
This was just so I would be able to test my changes until the CTFont migration gets merged in.  I expect these to be overwritten by your changes

---
In reply to: [78260160](https://github.com/Microsoft/WinObjC/pull/989#discussion_r78260160) [](ancestors = 78260160,78248109)